### ISSUE TITLE
Clean up py3.6 ascii_locale related tests and code

### ISSUE THREAD

**Issue by toofar** [qutebrowser/qutebrowser#7787]
The tests in test_invocations.py gated by the [ascii_locale](https://github.com/qutebrowser/qutebrowser/blob/d9e8b63/tests/end2end/test_invocations.py#L36) mark don't run anymore since we dropped py3.6 support.

Deleting the tests is easy, but is there any application code to clean up too?

**Comment by The-Compiler** [comment]
Good catch! See #3175, #3177 and 6989d4d2109ee40aad1c2567392dff2a1a324a7e.

I suppose in theory it's still a problem with UTF-8 anyways, just with a different set of characters:

```python
>>> b"\x80".decode('utf-8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte
```

makes me wonder if we could instead rewrite the tests to try that instead...
